api/v0/finder/model/provider_info.go

```diff
@@ -17,36 +14,19 @@ type ProviderInfo struct {
 	Publisher             *peer.AddrInfo     `json:",omitempty"`
 	IndexCount            uint64             `json:",omitempty"`
 	ExtendedProviders     *ExtendedProviders `json:",omitempty"`
+	FrozenAt              cid.Cid            `json:",omitempty"`
+	FrozenAtTime          string             `json:",omitempty"`
```

Comment:
This info is going to end up at the public facing `/providers` endpoint correct? 

I am curious what is the rationale for exposing these publicly?

Reply:
The reason for making this public is so that people know why that particular indexer does not return results for queries for content beyond the `FrozenAt` advertisement. For general provider information, as presented by indexstar, this would be filtered out.